cmd/minishift/cmd/config/config.go

```diff
@@ -120,6 +120,13 @@ var (
 	// Preflight values
 	CheckNetworkHttpHost = createConfigSetting("check-network-http-host", SetString, nil, nil, true)
 	CheckNetworkPingHost = createConfigSetting("check-network-ping-host", SetString, nil, nil, true)
+
+	// Network settings (Hyper-V only)
+	NetworkDevice = createConfigSetting("network-device", SetString, nil, nil, true)
+	IPAddress     = createConfigSetting("network-ipaddress", SetString, []setFn{validations.IsValidIPv4Address}, nil, true)
+	Netmask       = createConfigSetting("network-netmask", SetString, []setFn{validations.IsValidNetmask}, nil, true)
+	Gateway       = createConfigSetting("network-gateway", SetString, []setFn{validations.IsValidIPv4Address}, nil, true)
+	Nameserver    = createConfigSetting("network-nameserver", SetString, []setFn{validations.IsValidIPv4Address}, nil, true)
```

Comment:
Would like a slice to allow two nameservers, but this would need to be enforced in several places. So at the moment I think using a single option is OK, as this can be modified/improved in later commits

Reply:
@gbraad Do you already have a follow up issue for this?

Reply:
not specifically... but will do now...